ui/desktop/tests/e2e/pages/loginPage.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+/* eslint-disable no-undef */
+
+class LoginPage {
+  constructor(page) {
+    this.page = page;
+  }
+
+  async setClusterUrl(clusterUrl) {
+    await this.page.evaluate(() => localStorage.clear());
```

Comment:
It seems like I'm occasionally getting some failures here.

In some cases, it seems like the page loaded before the cluster url was cleared because it was showing the username/password screen. 

It one case, I got this error
```
1) tests/e2e/authentication.spec.js:13:3 › user/password authentication tests › Authenticates and signs out 

    Error: page.evaluate: Execution context was destroyed, most likely because of a navigation

       at tests/e2e/pages/loginPage.js:14

      12 |
      13 |   async setClusterUrl(clusterUrl) {
    > 14 |     await this.page.evaluate(() => localStorage.clear());
         |                     ^
      15 |     await this.page.getByLabel('Cluster URL').fill(clusterUrl);
      16 |     await this.page.getByRole('button', { name: 'Submit' }).click();
      17 |   }
```

Reply:
Also, it seems like something is not behaving correctly when using `npx playwright test --debug` (it opens to an authenticated state for me)

Reply:
Yeah that's the goal, it's to clear out the cluster url from localstorage (and any other state). I'll take a look to see if i can reproduce the occasional failure

Reply:
I moved the localstorage clearing to earlier on as part of the electron test fixture setup, and it seems to have fixed it while we were debugging.